styles/themes/teammember/_login-button.scss

```diff
@@ -0,0 +1,44 @@
+$login-button-font-size: 21px;
+$login-button-height: 30px;
+$login-button-padding: 0 34px;
+$login-button-container-height: 44px;
+$login-button-container-margin-top: 40px;
+$login-button-box-shadow-color: rgba(23, 54, 101, 0.8);
+
+
+.login-btn {
+    font-size: $login-button-font-size;
+    height: $login-button-container-height;
+    margin-top: $login-button-container-margin-top;
+}
+
+input,
+button {
+    &.login-button {
```

Comment:
I wanted to set a restriction for that class so that it applies to only those elements